packages/app/src/app/components/SubscribeForm/CheckoutForm/index.tsx

```diff
@@ -1,111 +1,104 @@
-import React from 'react';
-import { injectStripe, CardElement } from 'react-stripe-elements';
-import { Button } from '@codesandbox/common/lib/components/Button';
+import React, { useState } from 'react';
+import { injectStripe } from 'react-stripe-elements';
 import { logError } from '@codesandbox/common/lib/utils/analytics';
+import {
+  CardContainer,
+  Card,
+  NameInput,
+  ErrorText,
+  Label,
+  Submit,
+} from './elements';
+
+interface ICheckoutFormProps {
```

Comment:
@Saeris I think you should use a normal `type` instead of an `interface`

```suggestion
type CheckoutFormProps = {
```